modules/handlers/command_handlers.py

```diff
@@ -174,24 +176,51 @@ def reply_cmd(update: Update, context: CallbackContext):
                 chat_id=info['chat_id'],
-                text=
-                "Per mandare un messaggio ad un utente, rispondere al suo post con /reply seguito da ciò che gli si vuole dire"
+                text="La reply è vuota\n"\
+                "Per mandare un messaggio ad un utente, rispondere al suo post o report con /reply "\
+                    "seguito da ciò che gli si vuole dire"
             )
             return
-        try:
-            info['bot'].send_message(chat_id=user_id,
-                                     text="COMUNICAZIONE DEGLI ADMIN SUL TUO POST O REPORT:\n" + info['text'][7:].strip())
-            info['bot'].send_message(chat_id=info['chat_id'],
-                                 text="L'utente ha ricevuto il seguente messaggio:\n"\
-                                    "COMUNICAZIONE DEGLI ADMIN SUL TUO POST O REPORT:\n" + info['text'][7:].strip(),
-                                 reply_to_message_id=g_message_id)
-        except (BadRequest, Unauthorized) as e:
-            logger.warning("Notifying the user on /reply: %s", e)
+
+        g_message_id = update.message.reply_to_message.message_id
+
+        pending_post = PendingPost.from_group(group_id=info['chat_id'], g_message_id=g_message_id)
+        if pending_post is not None:  # the message was a pending post
+            try:
+                info['bot'].send_message(chat_id=pending_post.user_id,
+                                         text="COMUNICAZIONE DEGLI ADMIN SUL TUO ULTIMO POST:\n" +
+                                         info['text'][7:].strip())
+                info['bot'].send_message(chat_id=info['chat_id'],
+                                    text="L'utente ha ricevuto il seguente messaggio:\n"\
+                                        "COMUNICAZIONE DEGLI ADMIN SUL TUO ULTIMO POST:\n" + info['text'][7:].strip(),
+                                    reply_to_message_id=g_message_id)
+            except (BadRequest, Unauthorized) as e:
+                logger.warning("Notifying the user on /reply: %s", e)
+            return
+        report = Report.from_group(group_id=info['chat_id'], g_message_id=g_message_id)
+        if report is not None:  # the message was a report
+            try:
+                info['bot'].send_message(chat_id=report.user_id,
+                                         text="COMUNICAZIONE DEGLI ADMIN SUL TUO ULTIMO REPORT:\n" +
+                                         info['text'][7:].strip())
+                info['bot'].send_message(chat_id=info['chat_id'],
+                                    text="L'utente ha ricevuto il seguente messaggio:\n"\
+                                        "COMUNICAZIONE DEGLI ADMIN SUL TUO ULTIMO REPORT:\n" + info['text'][7:].strip(),
+                                    reply_to_message_id=g_message_id)
+            except (BadRequest, Unauthorized) as e:
+                logger.warning("Notifying the user on /reply: %s", e)
+            return
```

Comment:
this is like `return None` I think, so you could keep to use `return None` or just `return`, I prefer without None, but for `consistency` you should use one.